src/definitions/elements/emoji.less

```diff
@@ -0,0 +1,67 @@
+/*!
+ * # Fomantic UI - Emoji
+ * http://github.com/fomantic/fomantic-ui/
```

Comment:
Capitalize the repo and SSL
`http://github.com/fomantic/fomantic-ui/` => `https://github.com/fomantic/Fomantic-UI/`

Reply:
Will do